setup.py

```diff
@@ -254,16 +262,15 @@ def check_submodules():
         keywords='quantization, auto-tuning, post-training static quantization, post-training dynamic quantization, quantization-aware training, tuning strategy',
         license='Apache 2.0',
         url="https://github.com/intel/intel-extension-for-transformers",
-        ext_modules=[CMakeExtension(
-            "intel_extension_for_transformers.neural_engine_py", 'intel_extension_for_transformers/backends/neural_engine/')],
         packages=packages_list,
```

Comment:
you may also need change packages_list, otherwise you will also package neural engine compile python api to env

Reply:
thank you, Bo~